src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -304,6 +322,8 @@ public PreferenceSettings()
             ShowDetailedLayout = true;
             NamespacesToExcludeFromLibrary = new List<string>();
 
+            PythonTemplateFilePath = new PathManager(new PathManagerParams()).PythonTemplateFilePath;
```

Comment:
As far as I remember, when I wrote this I couldn't find a way to access any part of the current Dynamo Model instance.
You are right, a `PathManager` is instantiated as part of the `PreferenceSettings` class, which is created when making a `DynamoModel`, but since I couldn't get to it, resorted to the work-around of making a new dummy instance of `PathManager`. The static property will fix that, so this line will no longer be necessary.